src/qt/forms/sendcoinsdialog.ui

```diff
@@ -1248,6 +1248,16 @@
       </widget>
      </item>
      <item>
+      <widget class="QCheckBox" name="optInRBF">
+       <property name="text">
+        <string>Allow Replace-By-Fee</string>
+       </property>
+       <property name="toolTip">
+        <string>Signals that this transaction can be replaced with a transation paying higher fees (as long as the transaction is NOT confirmed).</string>
```

Comment:
"Indicates that the sender may wish to replace this transaction with a new one paying higher fees (prior to being confirmed)."